### ISSUE TITLE
[Feature]: Frame selected elements by shortcut

### ISSUE THREAD

**Issue by dimateos** [tldraw/tldraw#3909]
### What's the feature?

Sometimes I find myself selecting elements that I want to frame together (not group), but then when I start the framing tool all of them get deselected and I have to "reselect" them. Wouldn't it be simpler if they were simply framed (like when grouping)?

A few notes:
* There would be ambiguity between starting the frame tool and framing selected when you only have a few elements selected.
  * You dont want to force the user having to deselect all before being able to set the framinng, so I guess a custom shorcut could be used instead.
* I see CTRL+SHIFT+F as a hotkey to remove the framing (not listed among the "Keyboard shorcuts" tho). 
  * I would use this shorcuts too for framing (CTRL+F triggers browser search)
* When you select multiple frame elements (and no other kind) then CTRL+SHIFT+F unframes all, maybe the user wants to frame them together instead. Rare case but worth noting, here the user could simply be forced to frame them manually instead of by selection.
* This feature would allow framing elements while ignoring a few (later dragging the frame elsewhere), instead of having to frame all in a rect and them manually dragging some elements out.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by MitjaBezensek** [comment]
Yeah, like you mentioned it's quite hard to get all the use cases right. One distinction to groups is that frames do have a size of their own, so it felt like we should still allow the user to dictate that size. This is why we decided to go with an approach that you can frame all of those elements, you still have to use the frame tool though and drag over all of those elements like so:

https://github.com/user-attachments/assets/38292a6e-b9d9-4dd3-96d6-762268635c28

**Comment by dimateos** [comment]
Yep that is very convenient and it makes sense given the fact that frames must have an independent size.

I still believe my suggestion would be a worthwhile QoL feature.
It could have minimal priority tho, just giving ideas :)

* Basically the user selects the elements however they want / is used to: rect, shift+click, shift+rect, etc
* Then the hotkey would create a frame with the perfect size around them: no matter the size and perfectly aligned
  * An added benefit over drawing the shape would be that you can leave some elements unselected when you want them out of the frame, e.g. you are moving it elsewhere (not as common of a case I guess)

**Comment by MitjaBezensek** [comment]
Yeah, totally agree. The "frame only some shapes in this part of the canvas" doesn't really work with the current functionality. You have to move first, then frame.

Definitely makes sense for us to take a look if we can improve this. We already have the logic to fit the shapes in the frame ("Fit to content" option when you have the frame selected), so most of the pieces are already in place.

**Comment by steveruizok** [comment]
I'd be ok with adding a shortcut for this!

**Comment by dimateos** [comment]
Just in case I am unable to reopen once closed: /keepalive

**Comment by steveruizok** [comment]
I like the idea of a "frame selection" option. Figma has this and I use it all the time.
